content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -0,0 +1,214 @@
+
+Checking the box for :guilabel:`Timesheets` causes a new field to appear, labeled
+:guilabel:`Project`. This is the project where all the *timesheets* for this team's tickets will be
+recorded.  Click into the drop-down to select a :guilabel:`Project`. To create a new one, click
+into the drop-down and type a name, then click :guilabel:`Create`.
+
+.. image:: track_and_bill/track-bill-enable-settings.png
+   :align: center
+   :alt: View of a helpdesk team settings page emphasizing the track and bill time settings.
+
+Configure service products
+--------------------------
+
+When the :guilabel:`Time Billing` feature is enabled, a new product is created in the
+:guilabel:`Sales` app called :guilabel:`Service on Timesheets`. This product can be found under
+:menuselection:`Sales app --> Products --> Products`. Search for `Service on Timesheets`.
+
+This product is configured with the :guilabel:`Product Type` set to :guilabel:`Service` and the
+:guilabel:`Invoicing Policy` set to :guilabel:`Based on Timesheets`. This is the product that will
+be used when invoicing for services after they have been completed (*post-paid support services*).
+
+.. image:: track_and_bill/track-bill-product-based-on-timesheets.png
+   :align: center
+   :alt: View of a service product with the invoicing policy set to 'Based on timesheets'.
+
+In order to invoice for support services before the work has been completed (*prepaid support
+services*), a separate product with a different invoicing policy will need to be created.
+
+To create a new service product, go to :menuselection:`Sales app --> Products --> Products` and
+click :guilabel:`New`. Add a :guilabel:`Product Name`, and set the :guilabel:`Product Type` to
+:guilabel:`Service`. Set the :guilabel:`Invoicing Policy` to :guilabel:`Prepaid/Fixed Price`. This
+means that an invoice can be generated (and payment can be received) for this product *before*
+any timesheets entries have been recorded for these services.
+
+.. image:: track_and_bill/track-bill-product-prepaid-fixed.png
+   :align: center
+   :alt: View of a service product with the invoicing policy set to 'prepaid/fixed'.
+
+Set the :guilabel:`Sales Price`, and confirm the :guilabel:`Unit of Measure` is set to
+:guilabel:`Hours`.
+
+Invoice prepaid support services
+================================
+
+When support services are billed on a fixed price, an invoice can be created before any work is
+completed on the issue. In this case, a service product with the invoicing policy set to
+:guilabel:`Prepaid/Fixed Price` (like the one created above) would be used.
+
+Create a sales order with prepaid product
+-----------------------------------------
+
+To invoice a customer for prepaid support services, first create a sales order (SO) with the
+support services product. To do this, go to :guilabel:`Odoo Dashboard --> Sales --> Orders -->
+Quotations --> New`.
+
+Fill out the :guilabel:`Quotation` with the :guilabel:`Customer` information.
```

Comment:
You're absolutely right. My mistake. I meant to write "Quotation" as that's the first stage that a Sales Order is in. But, you're right - this is not an RfQ - I apologize for the error. I will go through my comments that incorrectly state that and delete them, so as to avoid any confusion.